bgpd/bgp_routemap.c

```diff
@@ -5708,6 +5760,36 @@ DEFPY_YANG(
 	return nb_cli_apply_changes(vty, NULL);
 }
 
+DEFPY_YANG(
+	match_community_limit, match_community_limit_cmd,
+	"[no$no] match community-limit [(0-65535)$limit]",
```

Comment:
We should do instead this way, and then checking for idx_limit is not needed.

```
"[no$no] match community-limit ![(0-65535)$limit]",
```

Reply:
ok